src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -476,10 +476,24 @@ public function __clone()
      */
     public function __toString()
     {
+        $protocol = '';
+        try {
+            $protocol = $this->server->get('SERVER_PROTOCOL');
+        } catch (\InvalidArgumentException $mallformedPath) {
+            trigger_error($mallformedPath->getMessage(), E_USER_ERROR);
```

Comment:
This is really edge case which will not happen until `get` is unchanged.

In general `trigger_error` shall spam php error log at least (ideally will be noticed).
In cases with raising exceptions by error handler - yes, it will be still fatal.

I'd propose to keep this, debugging this kind of issues normally takes too much time.